paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/BacsMandateConfirmationActivity.kt

```diff
@@ -0,0 +1,91 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.bacs
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.pm.ActivityInfo
+import android.os.Build
+import android.os.Bundle
+import androidx.activity.addCallback
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.LaunchedEffect
+import androidx.core.view.WindowCompat
+import com.stripe.android.common.ui.BottomSheet
+import com.stripe.android.common.ui.rememberBottomSheetState
+import com.stripe.android.paymentsheet.parseAppearance
+import com.stripe.android.uicore.StripeTheme
+import com.stripe.android.utils.AnimationConstants
+import kotlinx.coroutines.flow.collectLatest
+
+internal class BacsMandateConfirmationActivity : AppCompatActivity() {
```

Comment:
Do we have tests for this activity? Even if just E2E that's fine with me.

Reply:
Might be worth punting this until we have the E2E flow working, but regardless, there should be a test covering this code before we ship the feature.